source/_components/tradfri.markdown

```diff
@@ -20,6 +20,7 @@ For this to work, you need to install a modified lib-coap library:
 
 ```bash
 $ sudo apt-get install libtool
+$ sudo apt-get install autoconf
```

Comment:
Possibly.... I'm afraid I'm not too familiar with the requirements to agree/disagree

I'm just going by the steps I personally had to follow